tbd-fe/src/containers/Form/Form.test.js

```diff
@@ -57,4 +51,11 @@ describe("Form", () => {
     })
     expect(wrapper.state().radius).toEqual(newValue)
   })
+
```

Comment:
Does this test run? Looks like it's missing the expectation/assertion on it.

Reply:
my apologies. This was a test I started writing before I was asked to work on a different feature. I will remove it.